### PR TITLE
feat: implement revert for sd-boot

### PR DESCRIPTION
**Pull request by frezbo**
Implement revert for sd-boot.

### CODE REVIEW

**Comment by frezbo** [comment]
Also move the bootlabels back to grub, since it's only used there

File: internal/app/machined/pkg/runtime/v1alpha1/bootloader/mount/mount.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+			if errors.Is(err, blockdevice.ErrMissingPartitionTable) || os.IsNotExist(err) {
+				return nil
+			}
+
+			return err
+		}
+
+		probedBlockDevice = dev
+	case disk == "":
+		// attempt to probe BOOT partition on any disk
```
Comment (Unix4ever) [inline internal/app/machined/pkg/runtime/v1alpha1/bootloader/mount/mount.go:44]:
```suggestion
		// attempt to probe partition with partitionLabel on any disk
```

**Pull request merged**